src/sortedDict.jl

```diff
@@ -429,21 +122,21 @@ end
 
 function getkey{K,D,Ord <: Ordering}(m::SortedDict{K,D,Ord}, k_, default_)
     i, exactfound = findkey(m.bt, convert(K,k_))
-    exactfound? m.bt.data[i].k : convert(K,default_)
+    exactfound? m.bt.data[i].k : convert(K, default_)
 end
 
 ## Function delete! deletes an item at a given 
 ## key
 
-function delete!{K,D,Ord <: Ordering}(m::SortedDict{K,D,Ord}, k_)
-    i, exactfound = findkey(m.bt,convert(K,k_))
+@inline function delete!(m::SortedDict, k_)
+    i, exactfound = findkey(m.bt,convert(keytype(m),k_))
     !exactfound && throw(KeyError(k_))
     delete!(m.bt, i)
     m
 end
 
-function pop!{K,D,Ord <: Ordering}(m::SortedDict{K,D,Ord}, k_)
-    i, exactfound = findkey(m.bt,convert(K,k_))
+@inline function pop!(m::SortedDict, k_)
+    i, exactfound = findkey(m.bt,convert(keytype(m),k_))
```

Comment:
Same here.  I think you have the idea, so I'm going to stop commenting on these.